Sofa/framework/Simulation/Core/src/sofa/simulation/BaseSimulationExporter.cpp

```diff
@@ -95,6 +95,14 @@ void BaseSimulationExporter::handleEvent(Event *event){
             }
         }
     }
+    else if (AnimateBeginEvent::checkEventType(event))
```

Comment:
AnimateEndEvent or AnimageBeginEvent? 
Just asking as it is an export, not sure this make a difference.

Reply:
ok I think you already address that in  #4160  👍